test/e2e/storage/testpatterns/testpattern.go

```diff
@@ -324,6 +320,14 @@ var (
 		SnapshotDeletionPolicy: RetainSnapshot,
 		VolType:                DynamicPV,
 	}
+	// DynamicBlockSnapshotDelete is TestPattern for "Dynamic snapshot (block)"
+	DynamicBlockSnapshotDelete = TestPattern{
+		Name:                   "Dynamic Snapshot (block volmode)(delete policy)",
+		VolType:                DynamicPV,
+		VolMode:                v1.PersistentVolumeBlock,
+		SnapshotType:           DynamicCreatedSnapshot,
+		SnapshotDeletionPolicy: DeleteSnapshot,
+	}
```

Comment:
how about the Retain snapshot delete policy, and preprovision snapshot?

Reply:
I think delete policy and preprovision/dynamic should not impact block vs fs volumes, so I think it's fine to not include it so that we reduce the number of tests we run, unless you think a driver could have a bug with one of those particular combos.

Reply:
+1 to what Michelle mentioned.

Reply:
should be ok. but maybe manually test once just in case.